_topic_maps/_topic_map.yml

```diff
@@ -2314,6 +2314,8 @@ Topics:
   File: hcp-getting-started
 - Name: Managing hosted control planes
   File: hcp-managing
+- Name: Using feature gates on a hosted cluster
```

Comment:
The other heading here have `hosted control planes`, not `a hosted cluster`.  Just for consistency.

Reply:
Keeping it consistent with "hosted control planes"